share/vss/pedersen/vss_test.go

```diff
@@ -205,7 +205,7 @@ func TestVSSVerifierDecryptDeal(t *testing.T) {
 
 	// wrong dh key
 	goodDh := encD.DHKey
-	encD.DHKey = suite.Point()
+	encD.DHKey, _ = suite.Point().Null().MarshalBinary()
```

Comment:
always check error - just in case.

Reply:
If this line fails, then I should even panic, since that means our kyber library can't even marshal a "0" point it just created, and that means super big trouble, no ?